charts/pega/README.md

```diff
@@ -165,6 +167,7 @@ docker:
     url: "YOUR_DOCKER_REGISTRY"
     username: "YOUR_DOCKER_REGISTRY_USERNAME"
     password: "YOUR_DOCKER_REGISTRY_PASSWORD"
+  image_pull_secret_names: []
```

Comment:
Why do we want to take many secret names here?

Reply:
One use case would be different images provided in different registries.  For example, using a corporate managed busybox image in one registry and a Pega image located in a different registry specific to a project team or line of business.